Gemfile

```diff
@@ -13,7 +13,8 @@ gem "unicorn-worker-killer", "0.4.4"
 
 # Federation
 
-gem "diaspora_federation-rails", "0.2.0"
+gem "diaspora_federation-json_schema", "0.2.0", github: "diaspora/diaspora_federation", branch: "develop"
```

Comment:
We add some new gems to the gemfile here also. How do I split it?

I can do one commit with the version bump and then put all new gem additions to the https://github.com/diaspora/diaspora/pull/6726/commits/4ff4b297f748b3008a45258f5973f424b3d133c1 (Update the user data export archive format).